### PR TITLE
Add Brakeman static analysis

### PR DESCRIPTION
**Pull request by lozette**
Run Brakeman on pull requests to check for vulnerabilities. This workflow is taken directly from Github actions marketplace.

### CODE REVIEW

File: .github/workflows/brakeman-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    name: Brakeman Scan
+    runs-on: ubuntu-latest
+    steps:
+    # Checkout the repository to the GitHub Actions runner
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Customize the ruby version depending on your needs
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
```
Comment (pezholio) [inline .github/workflows/brakeman-analysis.yml]: I'm leaning towards using `ruby/setup-ruby` these days, as it takes the version of Ruby to use directly from the `.ruby-version` file

**Review by pezholio — APPROVED** [review]
Just one minor thing. Not a blocker, like

File: .github/workflows/brakeman-analysis.yml (hunk 1 above)
Comment (lozette) [reply .github/workflows/brakeman-analysis.yml]: Aha good call! Still learning how to do these actions 👍

Comment (pezholio) [reply .github/workflows/brakeman-analysis.yml]: Also, if you add

```
with:
        bundler-cache: true
```

to the action, it automagically caches the bundle, speeding up subsuqent builds - https://github.com/ruby/setup-ruby#caching-bundle-install-automatically

**Pull request merged**